src/data/markdown/docs/30 xk6-browser/02 API/03 BrowserType/launch--options--.md

```diff
@@ -13,12 +13,58 @@ Launches a new browser process.
+| disable-default-apps | `true` | Disables installation of default apps on first run. This is used during automated testing. |
+| disable-dev-shm-usage | `true` | The /dev/shm partition is too small in certain VM environments, causing Chrome to fail or crash. This flag provides a work-around for this issue (a temporary directory will always be used to create anonymous shared memory files). |
+| disable-extensions | `true` | Disables extensions. |
+| disable-features | ImprovedCookieControls,<br />LazyFrameLoading,<br />GlobalMediaControls,<br />DestroyProfileOnBrowserClose,<br />MediaRouter,<br />AcceptCHFrame | Comma-separated list of feature names to disable. |
+| disable-hang-monitor | `true` | Suppresses hang monitor dialogs in renderer processes. This may allow slow unload handlers on a page to prevent the tab from closing, but the Task Manager can be used to terminate the offending process in this case. |
+| disable-ipc-flooding-protection | `true` | Disables the IPC flooding protection. It is activated by default. Some javascript functions can be used to flood the browser process with IPC. This protection limits the rate at which they can be used. |
+| disable-popup-blocking | `true` | Disables pop-up blocking. |
+| disable-prompt-on-repost | `true` | Normally when the user attempts to navigate to a page that was the result of a post request, the browser prompts to make sure that's the intention of the user. This switch may be used to disable that check. This switch is used during automated testing. |
+| disable-renderer-backgrounding | `true` | Prevents renderer process backgrounding when set. |
+| force-color-profile | `srgb` | Forces all monitors to be treated as though they have the specified color profile. Accepted values are "srgb" and "generic-rgb" (currently used by Mac layout tests) and "color-spin-gamma24" (used by layout tests). |
+| metrics-recording-only | `true` | Enables the recording of metrics reports but disables reporting. This executes all the code that a normal client would use for reporting, except the report is dropped rather than sent to the server. This is useful for finding issues in the metrics code during UI and performance tests. |
+| no-first-run | `true` | Skips the "First Run" tasks, whether or not it's actually the first run, and the "What's New" page. This does not drop the "First Run" sentinel and thus doesn't prevent "First Run" from occurring the next time chrome is launched without this flag. It also does not update the last "What's New" milestone, so does not prevent "What's New" from occurring the next time chrome is launched without this flag. |
+| enable-automation | `true` | Enables indication that browser is controlled by automation. |
+| password-store | `basic` | Specifies which encryption storage backend to use. Possible values are kwallet, kwallet5, gnome, gnome-keyring, gnome-libsecret, basic. Any other value will lead to Chrome detecting the best backend automatically.  |
```

Comment:
```suggestion
| password-store | `basic` | Specifies which encryption storage backend to use. The possible values are kwallet, kwallet5, gnome, gnome-keyring, gnome-libsecret, and basic. Any other value will lead to Chromium detecting the best backend automatically. |
```

Reply:
I don't think the "the" is required, but applying for the other suggestions.